IpmiFeaturePkg/Include/Protocol/BmcAcpiSwChildPolicy.h

```diff
@@ -0,0 +1,31 @@
+/** @file
+  This protocol produces BmcAcpiSwChildPolicy Protocol.
+
+Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _BMC_ACPI_SW_CHILD_POLICY_H_

```

Comment:
```suggestion
#ifndef BMC_ACPI_SW_CHILD_POLICY_H_

```

https://tianocore-docs.github.io/edk2-CCodingStandardsSpecification/release-2.20/5_source_files/53_include_files.html#53-include-files

Include guards that start with _ or __ are reserved for compilers. Can you please remove the _?

Reply:
updated